lib/src/components/NxSmallThreatCounter/NxSmallThreatCounter.scss

```diff
@@ -67,9 +72,7 @@
       text-align: center;
       width: 100%;
 
-      &--zero {
-        visibility: visible;
-      }
+      @include counter-colors;
```

Comment:
Maybe add a comment that this is to override the zero styling

Reply:
Added here: [38e9f58](https://github.com/sonatype/sonatype-react-shared-components/pull/740/commits/38e9f5842c881638126d3c12fc863973a388c047)